runtime/vm/lookupmethod.c

```diff
@@ -214,6 +214,12 @@ processMethod(J9VMThread * currentThread, UDATA lookupOptions, J9Method * method
 
 	if (((lookupOptions & J9_LOOK_STATIC) && ((modifiers & J9AccStatic) == 0))
 	|| ((lookupOptions & J9_LOOK_VIRTUAL) && (modifiers & J9AccStatic))
+#if JAVA_SPEC_VERSION < 11 && defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+	/* Java 10- JVMS 6.5: if the resolved method is static or private, the invokeinterface instruction
+	 * throws an IncompatibleClassChangeError'
+	 */
+	|| ((lookupOptions & J9_LOOK_INTERFACE) && (lookupOptions & J9_LOOK_VIRTUAL) && (modifiers & J9AccPrivate))
```

Comment:
Macros should be used for the bit checks: `J9_ARE_ANY_BITS_SET`, `J9_ARE_ALL_BITS_SET` and `J9_ARE_NO_BITS_SET`. This function inconsistently incorporates different styles. It should be sufficient to only change the impacted `if` statement.
```suggestion
	|| (J9_ARE_ALL_BITS_SET(lookupOptions, J9_LOOK_INTERFACE | J9_LOOK_VIRTUAL) && J9_ARE_ALL_BITS_SET(modifiers, J9AccPrivate))
```